cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -79,21 +76,22 @@ describe(["@tier1"], "Source Analysis", () => {
         Application.open(100, true);
     });
 
-    it("Source + dependencies analysis on tackletest app", function () {
+    it("Source + dependencies analysis on book-server app, validating issues on single app", function () {
```

Comment:
Why are we changing this test ? 
The test you want is in file source_analysis_without_cred.test.ts

Reply:
https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis_without_creds.test.ts#L44

Reply:
Reverted this change, moved validation to required test